jest/setupEnv.js

```diff
@@ -41,3 +42,22 @@ RequestUtil.json = function () {};
     };
   }
 }());
+
```

Comment:
I like the separate IIFE for clarity, but can we add a short explanation of what this block is doing?